### PR TITLE
Initial scaffolding

### PR DESCRIPTION
**Pull request by sglanzer-fire**
# minor# initial commit

### CODE REVIEW

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-blueplanet/ember-block-slots/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-blueplanet/ember-block-slots)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+language: node_js
```
Comment (sandersky) [inline .travis.yml:1]: Mind updating this file based on https://github.com/ciena-blueplanet/ember-prop-types/blob/master/.travis.yml

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "ember-block-slots",
+  "dependencies": {
+    "ember": "~2.4.1",
+    "ember-cli-shims": "0.1.0",
+    "ember-cli-test-loader": "0.2.2",
+    "ember-mocha-adapter": "~0.3.1",
+    "ember-qunit-notifications": "0.1.0",
```
Comment (sandersky) [inline bower.json:8]: Is this relevant when using mocha?

File: tests/helpers/module-for-acceptance.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { module } from 'qunit'
```
Comment (sandersky) [inline tests/helpers/module-for-acceptance.js:1]: I believe we can just remove this file as it depends on qunit which we aren't using.

**Comment by sglanzer-fire** [comment]
@sandersky Sorry, missed your comments - I'll PR in those changes

**Pull request merged**